### PR TITLE
Libxml rebuild

### PR DESCRIPTION
**Pull request by SC426**
### Links

- [PKG-4629](https://anaconda.atlassian.net/browse/PKG-4629)

[PKG-4629]: https://anaconda.atlassian.net/browse/PKG-4629?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ test:
 
 about:
   home: https://github.com/xkbcommon/libxkbcommon
-  license: MIT/X11 Derivative
+  license: X11
```
Comment (JamesRobertsonGames) [inline recipe/meta.yaml]: Not sure about this change, the license seems to be derivative still?

https://github.com/xkbcommon/libxkbcommon/blob/master/LICENSE

> All licenses are derivative of the MIT/X11 license, mostly identical other
than no-endorsement clauses (e.g. paragraph 4 of The Open Group's license).

Comment (skupr-anaconda) [reply recipe/meta.yaml]: I think the correct license is `MIT AND X11`

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: `MIT AND X11` is more correct indeed. Though it's a little bit complicated because each license is slightly modified as far as I see. Debian, Gentoo and Arch Linux say it's MIT while Fedora says it's `MIT AND X11 AND MIT-CMU`.

I feel like it has to do with different components. For example the data files are licensed differently, so if you ship the data files separately, it will affect the package license.

In other words, I don't know what is the real correct license without doing a deep dive into he source code and the licenses themselves.

**Review by skupr-anaconda — APPROVED** [review]
